### PR TITLE
config: change tidb_top_sql_agent_address to config top-sql.receiver-address

### PR DESCRIPTION
**Pull request by crazycs520**
…
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

In order to easier to enable `TopSQL`, I change the original  instance variable `tidb_top_sql_agent_address` to the config `top-sql.receiver-address`. Since it's very inconvenient to connect each TiDB server and set the instance variable.

### What is changed and how it works?

* remove `tidb_top_sql_agent_address` variable.
* add `top-sql.receiver-address` config.

### Check List 

Tests 

- [x] Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by crazycs520** [comment]
@breeswish @dragonly PTAL

File: config/config.go [hunk 1]
```diff
@@ -537,6 +538,11 @@ type StmtSummary struct {
 	HistorySize int `toml:"history-size" json:"history-size"`
 }
 
+// TopSQL is the config for TopSQL.
+type TopSQL struct {
+	ReceiverAddress string `toml:"receiver-address" json:"receiver-address"`
```
Comment (breeswish) [inline config/config.go:543]: Please add comments about what is a receiver address.

File: util/topsql/reporter/reporter.go [hunk 2]
```diff
@@ -552,7 +553,7 @@ func (tsr *RemoteTopSQLReporter) doReport(data reportData) {
 		return
 	}
 
-	agentAddr := variable.TopSQLVariable.AgentAddress.Load()
+	agentAddr := config.GetGlobalConfig().TopSQL.ReceiverAddress
```
Comment (breeswish) [inline util/topsql/reporter/reporter.go:556]: Does this mean that we will not allow dynamically changing it at all?

**Review by breeswish — COMMENTED** [review]
The rest LGTM

File: util/topsql/reporter/reporter.go (hunk 2 above)
Comment (crazycs520) [reply util/topsql/reporter/reporter.go:556]: Yes.

**Review by dragonly — APPROVED** [review]
LGTM

**Review by breeswish — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**